src/it/java/com/google/checkstyle/test/base/AbstractGoogleModuleTestSupport.java

```diff
@@ -110,4 +139,71 @@ protected static List<Configuration> getModuleConfigsByIds(String... moduleIds)
         return getModuleConfigsByIds(CONFIGURATION, moduleIds);
     }
 
+    /**
+     * Performs verification of the file with the given file path against the whole config.
+     *
+     * @param filePath file path to verify.
+     * @throws Exception if exception occurs during verification process.
+     */
+    protected void verifyWithWholeConfig(String filePath) throws Exception {
+        verifyWithItConfig(CONFIGURATION, filePath);
+    }
+
+    // until https://github.com/checkstyle/checkstyle/issues/14937
```

Comment:
Assuming this method will be removed upon migration completion, I'm fine to keep its logic as it is now.

Reply:
This method might stay, we will use it for complicated input files for which following all googles rules would be difficult.

Reply:
This would still go back to why don't we go against the entire config which started us on this path. I understand not everything can be done in the short time of GSoC, but even complicated input files should be converted eventually.

Reply:
Yess, In GSoC time period I'm not sure if I can convert all tests to use whole config as I have other issues to work on too. But after GSoC, I can complete it.